src/odemis/driver/tescan.py

```diff
@@ -873,11 +871,13 @@ def _pollVAs(self):
                         self.probeCurrent._value = new_pc
                         self.probeCurrent.notify(new_pc)
 
-                    bmode = self.parent._device.ScGetBlanker(1)
-                    blanked = (bmode != 0)
-                    if blanked != self.blanker._value:
-                        self.blanker._value = blanked
-                        self.blanker.notify(blanked)
+                    # if blanker is in auto, don't change its value
+                    if not self.blanker.value is None:
```

Comment:
is this different then: `if self.blanker.value is not None:`? If it is not different, I think `if self.blanker.value is not None:` is more readable.

Reply:
Although in my mind the version I wrote feels more natural, I agree that the one you propose is generally more readable (as it's closer to the English grammar).
I'll change that.